src/edfi-performance-test/edfi_performance_test/api/client/ed_fi_api_client.py

```diff
@@ -96,7 +95,7 @@ def __init__(self, client: HttpSession, token: str = "", api_prefix: str = ""):
                 # Default to FooClient => foo_client
                 subclient_name = _title_case_to_snake_case(subclient_class.__name__)
 
-            subclient = subclient_class(client, token=token)
+            subclient = subclient_class(client, token=self.token)
```

Comment:
Again, this was using the argument and ignoring the value set by the constructor.